scripts/helper_scripts/v2dl3_qsub.py

```diff
@@ -0,0 +1,87 @@
+"""
+Submit v2dl3 EventDisplay jobs to batch farm. Uses input from script that contains induvidual commands per line.
+The required script is created using the script ANALYSIS.anasum_parallel_from_runlist_v2dl3.sh.
+"""
+
+from datetime import date
+import os
+import subprocess
+
+import click
+
+import pyV2DL3
+
+# get the directory of the run_loop.py file
+dirname = os.path.dirname(pyV2DL3.__file__)
+# get today's data for the log dir
+today = date.today().strftime("%y%m%d")
+
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--v2dl3_script",
+    type=click.Path(exists=True),
+    help="Script that contains individual commands per line that can also be used"
+    " in stand alone mode with the v2dl3 converter",
+)
+@click.option(
+    "--conda_env",
+    default="v2dl3",
+    help='Name of the conda environment. (Default: "v2dl3")',
+)
+@click.option(
+    "--conda_exe",
+    default="$CONDA_EXE",
+    help="Path to the conda executable. Change if required.",
+)
+@click.option(
+    "--rootsys", default="$ROOTSYS", help="Path of the ROOTSYS. Change if required."
+)
+@click.option("--output_dir", default="", help="output directory of fits files")
+@click.option("--add_option", default="", help="Option to add when running v2dl3.")
+def cli(v2dl3_script, conda_env, conda_exe, rootsys, output_dir, add_option):
+    with open(v2dl3_script, "r") as script:
+        commands = [
+            line for line in script.read().splitlines() if line.startswith("v2dl3")
```

Comment:
Note that this script will need some adjustments to work with the changes in the `V2DL3SCRIPT`. For example, this line will not work anymore! Not really clear what it should be as it uses the user-specific `$V2DL3` now:

https://github.com/VERITAS-Observatory/Eventdisplay_AnalysisScripts_VTS/blob/91ec7d62d2fc972e1a81d036ac33d7c481fe64ad/scripts/ANALYSIS.anasum_parallel_from_runlist.sh#L306

I still think the easiest would be to install v2dl3 as a package and have a console script `v2dl3-eventdisplay`. This would remove the need for the `$V2DL3` variable I believe?